geomesa-nifi-processors/geomesa-nifi-processors-fs/pom.xml

```diff
@@ -46,33 +46,11 @@
             <artifactId>geomesa-convert-all_2.11</artifactId>
         </dependency>
 
-        <dependency>
-            <groupId>org.apache.zookeeper</groupId>
-            <artifactId>zookeeper</artifactId>
-        </dependency>
-
-        <dependency>
```

Comment:
i think we need these hadoop files, as the fsds uses the hadoop api. did you try the fs nar after these changes? we probably don't need zookeeper, but let me double check that.